code/mod_table/mod_table.cpp

```diff
@@ -794,6 +795,21 @@ void parse_mod_table(const char *filename)
 			}
 		}
 
+		if (optional_string("$FRED Briefing window resolution:")) {
+			int res[2];
+			if (stuff_int_list(res, 2) == 2) {
+				Briefing_window_resolution[0] = res[0];
+				Briefing_window_resolution[1] = res[1];
+				mprintf(("Game Settings Table: Setting FRED briefing window resolution from (%ix%i) to (%ix%i)\n",
+					res[0],
+					res[1],
```

Comment:
This logging statement will show two sets of the same resolution, not the old and new resolution.  You would need to swap the logging arguments and move the assignment to after the logging.

Reply:
Not anymore